modules/orca/main.tf

```diff
@@ -11,14 +11,15 @@ terraform {
 
 ## AWS Provider Settings
 provider "aws" {
-  profile = var.aws_profile
```

Comment:
do we have to change this `aws_profile` var in the [main.tf](https://github.com/nasa/cumulus-orca/blob/develop/main.tf#L15) as well?

Reply:
No. This is a local variable.